internal/handlers/handlers.go

```diff
@@ -1 +1,84 @@
 package handlers
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+const formFieldName = "myFile"
+
+// IndexHandler возвращает HTML-страницу index.html по GET-запросу на корневой эндпоинт.
+//
+// Используется для отображения формы загрузки файла.
+// Файл index.html должен находиться в корне проекта.
+//
+// Параметры запроса:
+//   - GET /
+//
+// Ответ клиенту:
+//   - HTTP 200 OK с HTML-контентом.
+//   - HTTP 404 или другая ошибка, если файл не найден.
+func IndexHandler(w http.ResponseWriter, r *http.Request) {
+	path := filepath.Join(".", "index.html")
+	http.ServeFile(w, r, path)
```

Comment:
да можно не усложнять
```suggestion
	http.ServeFile(w, r, "./index.html")
```